src/components/PageTitle.jsx

```diff
@@ -25,8 +25,8 @@ const PageTitle = ({ children, ...rest }) => {
     <>
       {!isRoot && (
         <BarLeft>
-          <IconButton onClick={navigateBack} size="large">
-            <Icon color="secondary" icon={LeftIcon} />
+          <IconButton onClick={navigateBack} size="large" title="Previous">
```

Comment:
Nice for the A11y! 

nit: should be translated ;)

Reply:
updated